.vscode/extensions.json

```diff
@@ -1,6 +1,3 @@
 {
-  "recommendations": [
-    "arcanis.vscode-zipfs",
-    "dbaeumer.vscode-eslint"
-  ]
+  "recommendations": []
```

Comment:
This also seems unrelated to this particular problem?

Reply:
It's unrelated but it's to silence some popups when opening vscode. I can make a separate PR if you prefer, but it's quite a tiny change with no impact.